data/registers/adc_l111x.yaml

```diff
@@ -0,0 +1,1309 @@
+  - name: MEMRESIFG23
+    description: MEMRES23 data loaded interrupt.
+    value: 32
+enum/ENDADD:
+  bit_size: 5
+  variants:
+  - name: ADDR_00
+    description: MEMCTL0 is selected as end address of sequence.
+    value: 0
+  - name: ADDR_01
+    description: MEMCTL1 is selected as end address of sequence.
+    value: 1
+  - name: ADDR_02
+    description: MEMCTL2 is selected as end address of sequence.
+    value: 2
+  - name: ADDR_03
+    description: MEMCTL3 is selected as end address of sequence.
+    value: 3
+  - name: ADDR_04
+    description: MEMCTL4 is selected as end address of sequence.
+    value: 4
+  - name: ADDR_05
+    description: MEMCTL5 is selected as end address of sequence.
+    value: 5
+  - name: ADDR_06
+    description: MEMCTL6 is selected as end address of sequence.
+    value: 6
+  - name: ADDR_07
+    description: MEMCTL7 is selected as end address of sequence.
+    value: 7
+  - name: ADDR_08
+    description: MEMCTL8 is selected as end address of sequence.
+    value: 8
+  - name: ADDR_09
+    description: MEMCTL9 is selected as end address of sequence.
+    value: 9
+  - name: ADDR_10
+    description: MEMCTL10 is selected as end address of sequence.
+    value: 10
+  - name: ADDR_11
+    description: MEMCTL11 is selected as end address of sequence.
+    value: 11
+  - name: ADDR_12
+    description: MEMCTL12 is selected as end address of sequence.
+    value: 12
+  - name: ADDR_13
+    description: MEMCTL13 is selected as end address of sequence.
+    value: 13
+  - name: ADDR_14
+    description: MEMCTL14 is selected as end address of sequence.
+    value: 14
+  - name: ADDR_15
+    description: MEMCTL15 is selected as end address of sequence.
+    value: 15
+  - name: ADDR_16
+    description: MEMCTL16 is selected as end address of sequence.
+    value: 16
+  - name: ADDR_17
+    description: MEMCTL17 is selected as end address of sequence.
+    value: 17
+  - name: ADDR_18
+    description: MEMCTL18 is selected as end address of sequence.
+    value: 18
+  - name: ADDR_19
+    description: MEMCTL19 is selected as end address of sequence.
+    value: 19
+  - name: ADDR_20
+    description: MEMCTL20 is selected as end address of sequence.
+    value: 20
+  - name: ADDR_21
+    description: MEMCTL21 is selected as end address of sequence.
+    value: 21
+  - name: ADDR_22
+    description: MEMCTL22 is selected as end address of sequence.
+    value: 22
+  - name: ADDR_23
+    description: MEMCTL23 is selected as end address of sequence.
+    value: 23
+enum/EVT1_CFG:
+  bit_size: 2
+  variants:
+  - name: DISABLE
+    description: The interrupt or event line is disabled.
+    value: 0
+  - name: SOFTWARE
+    description: The interrupt or event line is in software mode. Software must clear the RIS.
+    value: 1
+  - name: HARDWARE
+    description: The interrupt or event line is in hardware mode. The hardware (another module) clears automatically the associated RIS flag.
+    value: 2
+enum/FPUB_1_CHANID:
```

Comment:
All CHANID enums should be deleted. Otherwise you get an enum with a single value which is unusable without transmute.